_studio/mfx_lib/shared/src/mfx_h264_encode_vaapi.cpp

```diff
@@ -2753,6 +2809,10 @@ mfxStatus VAAPIEncoder::Execute(
                                                           m_vaContextEncode, m_maxFrameSizeId), MFX_ERR_DEVICE_FAILED);
     configBuffers[buffersCount++] = m_maxFrameSizeId;
 
+    MFX_CHECK_WITH_ASSERT(MFX_ERR_NONE == SetMultiPassFrameSize(task, fieldId, m_vaDisplay,
+                                                          m_vaContextEncode, m_multiPassFrameSizeId), MFX_ERR_DEVICE_FAILED);
+    configBuffers[buffersCount++] = m_multiPassFrameSizeId;
```

Comment:
this code traditionally uses dangerous way to store buffers in vector. Could please check that initial resize of vector takes into account current buffer? I mean that your change won't cause out of bounds writing. See line https://github.com/Intel-Media-SDK/MediaSDK/blob/373f2a0250590e51ebe28b04bfdb4b290c1f1671/_studio/mfx_lib/shared/src/mfx_h264_encode_vaapi.cpp#L2088

Reply:
Will add one for MAX_CONFIG_BUFFERS_COUNT macro.